content/en/platform/corda/4.8/enterprise/release-notes-enterprise.md

```diff
@@ -17,6 +17,33 @@ weight: 1
 
 # Corda Enterprise release notes
 
+## Corda Enterprise 4.8.2
+
+Corda Enterprise 4.8.2 is a patch release of Corda Enterprise that fixes security vulnerabilities in Corda Enterprise 4.8 and 4.8.1, and offers greater compatibility with recent versions of FutureX.
+
+### Upgrade recommendation
+
+As a developer, you should upgrade to the [latest released version of Corda](../../../../../en/platform/corda/4.8/enterprise.html) as soon as possible. The latest Corda Enterprise release notes are on this page, and you can find the latest upgrade guide [here](../../../../../en/platform/corda/4.8/enterprise/upgrading-index.md).
+
+As a node operator, you should upgrade to the [latest released version of Corda](../../../../../en/platform/corda/4.8/enterprise.html) if the fixed issues listed below are relevant to your work.
+
+### Fixed issues
```

Comment:
Did we want to include that ENT 4.8 now supports Cockroach 21.2.8? 
https://r3-cev.atlassian.net/browse/ENT-6334

Did we include something for this in the end? 
https://r3-cev.atlassian.net/browse/NAAS-295

Reply:
actually 21.1.7 (long story) but sorted now.